src/proxy-ui-api/frontend/src/views/Clients/LocalGroups/LocalGroups.vue

```diff
@@ -46,28 +46,30 @@
       >
     </div>
 
-    <v-card flat>
-      <table class="xrd-table details-certificates">
-        <tr>
-          <th>{{ $t('localGroups.code') }}</th>
-          <th>{{ $t('localGroups.description') }}</th>
-          <th>{{ $t('localGroups.memberCount') }}</th>
-          <th>{{ $t('localGroups.updated') }}</th>
-        </tr>
-        <template v-if="groups && groups.length > 0">
-          <tr v-for="group in filtered()" :key="group.code">
-            <td class="identifier-wrap">
-              <span class="cert-name" @click="viewGroup(group)">{{
-                group.code
-              }}</span>
-            </td>
-            <td class="identifier-wrap">{{ group.description }}</td>
-            <td>{{ group.member_count }}</td>
-            <td>{{ group.updated_at | formatDate }}</td>
-          </tr>
-        </template>
-      </table>
-    </v-card>
+    <v-data-table
```

Comment:
This table has a placeholder text of `No data available`. Should it be changed to `No local groups` or something? I think the same goes with Service Clients in the Service Clients tab

Reply:
Fixed these.